index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Final Trial</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+
+
+
+
+    <section id="welcome-section">
+     <a href="#projects"> <nav id="navbar">Go to Projects</nav></a>
+<h1 id="title">Hey were are Billal and Kevin</h1>
+ </section>
+    <section id="projects">
+      <div class="project-tile">
+  <a href="google.com" target="_blank"><p>Sample project</p></a> 
+      </div>
+     <a id="profile-link" href="https://github.com/bill7pearl/final-trial" target="_blank"> <p>My portfolio</p></a>
+    </section>
+
+
+
+    <div id="header">
+      <ul>
+        <li>Contact</li>
+        <li>Work</li>
+        <li>About</li>
```

Comment:
- [ ] Good job for using the Git hub-inline feature that fits the context the comment is about, But Kindly I recommend you to phrased the comments politely, as suggestions instead of orders and always includes an explanation, a "why", for each requested change.
- For Example : - 
  "Kindly I recommend you to use `a` tag for each navigation bar Apart from being semantically correct, the` a` tag will let you connect your social networks with the icons, using `href`. "